docs/_includes/getting-started/download.html

```diff
@@ -42,11 +42,11 @@ <h2 id="download-cdn">Bootstrap CDN</h2>
 
   <h2 id="download-bower">Install with Bower</h2>
   <p>You can also install and manage Bootstrap's Less, CSS, JavaScript, and fonts using <a href="http://bower.io">Bower</a>:</p>
-  {% highlight bash %}$ bower install bootstrap{% endhighlight %}
+  {% highlight bash %}bower install bootstrap{% endhighlight %}
```

Comment:
Will this actually get published to Bower or do we need to pull/modify this?

Reply:
Ah, good point. No idea about this, I just replaced the `$` symbol. Your call. If so, I will remove the references in another PR when I'll backport more stuff.